src/tezos-client/signers/in-memory.ts

```diff
@@ -0,0 +1,10 @@
+import { InMemorySigner } from "@taquito/signer";
+import { readFile } from "fs/promises";
+import { WALLET_PRIVATE_KEY_FILE } from "src/utils/constants";
+
+export const getInMemorySigner = async (key?: string) => {
```

Comment:
Can we throw an error here if the private key is non-existent or invalid?

Reply:
No need. We have global error handler and taquito wont let you create invalid signer. Output for invalid setup with current code is:
```sh
# missing payout_wallet_private.key
ENOENT: no such file or directory, open 'payout_wallet_private.key'
# invalid key
The key  is invalid. Unsupported key type
```